eclair-core/src/test/scala/fr/acinq/eclair/transactions/TransactionsSpec.scala

```diff
@@ -234,20 +241,20 @@ class TransactionsSpec extends FunSuite with Logging {
 
     {
       // local spends delayed output of htlc1 timeout tx
-      val claimHtlcDelayed = makeClaimDelayedOutputTx(htlcTimeoutTxs(0).tx, localDustLimit, localRevocationPriv.publicKey, toLocalDelay, localDelayedPaymentPriv.publicKey, finalPubKeyScript, feeratePerKw)
+      val claimHtlcDelayed = makeClaimDelayedOutputTx(htlcTimeoutTxs(1).tx, localDustLimit, localRevocationPriv.publicKey, toLocalDelay, localDelayedPaymentPriv.publicKey, finalPubKeyScript, feeratePerKw)
```

Comment:
Note that these are HTLC 2nd stage transactions not HTLC outputs. The order changed because we now generate the 2nd stage transactions sorted in the same way we sort the HTLC outputs in the commitment transaction, and this wasn't the case before (details at `Transactions.makeHtlcTxs`)

Reply:
Got it, nothing to worry about then